sections/projects.md

```diff
@@ -170,6 +182,8 @@
 | rate_card                    | Integer, ID of rate card                                                                          |
 | remaining_auto_calculated    | Boolean (Defaults to true)                                                                        |
 | use_project_allocations      | Boolean, deprecated. Uses company setting instead.                                                |
+| use_baseline                 | Boolean (Defaults to false)                                                                       |
+| baseline_win_chance          | Double (Between 0.0 and 1.0) (Defaults to 1.0)                                                    |
```

Comment:
Not possible to set the target minutes in the creation?
Good description. Put the "(Between 0.0 and 1.0)" on the other places as well

Reply:
I added this to creation, along with some extra documentation on baseline_target_minutes and budget. Not sure if it is the best solution, but I needed a way to highlight that the fields are impacted by the budget_type you have selected.